lib/timelines/models/concerns/has_events.rb

```diff
@@ -0,0 +1,11 @@
+require "active_support/concern"
+
+module Timelines
```

Comment:
Have I put this in the right folder? Does it need to be inside `/models`? That's how it was structured in the main insights repo.

Reply:
Yep, all good 👍